index.html

```diff
@@ -7,10 +7,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Portfolio Site</title>
     <link rel="stylesheet" href="./style.css">
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
 </head>
 
 <body>
+    <nav>
+        <div class="menu">
+            <div>
+                <img id="triangle" src="./assets/img/triangle.png" alt="Triangle">
+                <img id="square" src="./assets/img/square.png" alt="square">
+            </div>
+            <div>
+                <img id="menuimg" src="./assets/img/menu.png" alt="square">
+            </div>
+        </div>
+    </nav>
```

Comment:
- [ ] The container of the toolbar section should be wrapped with a `header` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Kindly add a `header` tag around the `nav` tag.